types/johnny-five/index.d.ts

```diff
@@ -84,6 +84,168 @@ export class Altimeter {
     on(event: 'data', cb: (data: any) => void): this;
 }
 
+export type EasingFnVariant =
+    | 'linear'
+    | 'inQuad'
+    | 'outQuad'
+    | 'inOutQuad'
+    | 'inCube'
+    | 'outCube'
+    | 'inOutCube'
+    | 'inQuart'
+    | 'outQuart'
+    | 'inOutQuart'
+    | 'inQuint'
+    | 'outQuint'
+    | 'inOutQuint'
+    | 'inSine'
+    | 'outSine'
+    | 'inOutSine'
+    | 'inExpo'
+    | 'outExpo'
+    | 'inOutExpo'
+    | 'inCirc'
+    | 'outCirc'
+    | 'inOutCirc'
+    | 'inBack'
+    | 'outBack'
+    | 'inOutBack'
+    | 'inBounce'
+    | 'outBounce'
+    | 'inOutBounce';
+export type WithEasing<T> = T & {
+    /**
+     * An easing function from ease-component to apply to the tweened value of the previous and next keyFrames.
+     * See the {@link https://www.npmjs.com/package/ease-component ease-component documentation} for a list of available easing functions (default: "linear")
+     */
+    easing?: EasingFnVariant;
+};
+
+/**
+ * If a single device is being animated, keyFrames may be a single dimensional array.
+ *
+ * If more than one device is being animated it must be 2-dimensional (an array of arrays).
+ * We call this a "keyFrame set". The index of each device in the target maps to the same index in a keyFrame set so the length of the two should be identical.
+ * Each keyFrame array should have an element that maps to each cue point in the cuePoints array.
+ * keyFrames[0][0] for example represents the position of the first device in your animation target at the first cuePoint.
+ *
+ * These may or may not be the same length. If there are fewer elements in a keyFrame array than in the cuePoints array, then the keyFrame array will be right padded with null values.
+ *
+ * Elements in a keyFrame array represent a device's position at the corresponding cuePoint. Positions can be described in a number of ways:
+ */
+export type KeyframeValue =
+    | number
+    | null
+    | false
+    | WithEasing<{
+          /** A step in degrees from the previous cuePoint position. */ step: number;
```

Comment:
Is there a reason the comments are on the same line as the code for these types?